meta.yaml

```diff
@@ -1,26 +1,25 @@
-{% set version = "2020.7.30.1" %}
+{% set version = "2021.07.0" %}
 
 package:
   name: geocat-viz
   version: {{ version }}
 
 source:
```

Comment:
This conda builds might be aimed at either debugging or actual release, so it mwould be great to put descriptive comments for future reference. Something similar to:

```
#git_rev: main   # For building from main branch for debugging purposes
git_tag: v{{ version }}   # For building from a specific tag
git_url: https://github.com/ncar/geocat-viz.git
#url: https://github.com/NCAR/geocat-viz/archive/v{{ version }}.tar.gz   # For building from a specific tag's tarball
```

FYI: Using `git_tag` plus `git_url` above gives us the same way of releasing our actual versions (to be uploaded into conda) as we do in GeoCAT-comp or GeoCAT-f2py, where we first "tag" our `main` branch as a release at any point in the repo, then we generate our conda package from that "tagged" release with this `meta.yaml` file. On the other hand, the developer could anytime use `git_rev` plus `git_url` or only `url` itself for debugging purposes in order to generate a test conda package from any branch or specific tagged release, respectively. We can discuss the details further if you wish.

Reply:
Oh! I had no idea this was a thing, thanks! I've put your suggestion in.